opentreemap/exporter/tasks.py

```diff
@@ -24,6 +27,8 @@
 from exporter.user import write_users
 from exporter.util import sanitize_unicode_record
 
+_UDFC_FIELDS = tuple(['udf:' + name for name in UDFC_NAMES])
```

Comment:
FYI, while you do need the outer `tuple( ... )` there is no need for the inner `[ ... ]` here.